Makefile

```diff
@@ -38,9 +38,9 @@ FFTW_LIBRARY       =
 
 # GNU Scientific library flags and paths.  If paths are empty, tries standard default linux paths.
 #--------------------------------------------------------------------------------------------------
-GSL                = 0
+GSL                = 1
 GSL_INCLUDE        =
-GSL_LIBRARY        =
+GSL_LIBRARY        = 
```

Comment:
No, do like in the icsilcc branch where it happens only on Travis. we do NOT want to have GSL on by default